index.html

```diff
@@ -13,140 +13,188 @@
     <header>
         <nav class="flex-nav"> 
             <a href="#" class="toggleNav">☰</a>
+            <ul class="menu">
+                <li>
+                    Hello
+                    <div class="line-hello"></div>
+                </li>
+                <li>Portfolio
+                    <div class="line-portfolio"></div>
+                </li>
+                <li>About
+                    <div class="line-about"></div>
+                </li>
+                <li>Contact
+                    <div class="line-contact"></div>
+                </li>
+            </ul>
```

Comment:
[OPTIONAL]
I strongly suggest you make each link an `a` tag as this is best practice